ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/config/ExecutorsConfig.java

```diff
@@ -0,0 +1,16 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+@Configuration
+public class ExecutorsConfig {
+
+	@Bean
+	public ExecutorService executorService() {
+		return Executors.newCachedThreadPool();
```

Comment:
Don't know how an Executor works, but i'd expect a pool size here as an extra (minor) performance improvement.

Reply:
Thought about it. Maybe something for in config?